packages/storage-documents/src/privacy-scan-types/privacy-page-scan-batch-request.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { StorageDocument } from '../storage-document';
+import { ItemType } from '../item-type';
+import { ReportGroupRequest, WebsiteRequest } from '../on-demand-page-scan-batch-request';
+import { PrivacyScanConfiguration } from './privacy-scan-configuration';
+
+/**
+ * The client page scan run batch request document.
+ */
+export interface PrivacyPageScanBatchRequest extends StorageDocument {
+    itemType: ItemType.privacyPageScanBatchRequest;
+    scanRunBatchRequest: PrivacyScanRunBatchRequest[];
+    privacyScanConfig: PrivacyScanConfiguration;
+}
+
+export interface PrivacyScanRunBatchRequest {
+    scanId: string;
+    url: string;
+    site?: WebsiteRequest;
+    priority: number;
+    reportGroups?: ReportGroupRequest[];
+}
```

Comment:
no need for this type, we will use existing structure with `privacyScan?: boolean;` flag